server/opts_test.go

```diff
@@ -2460,3 +2462,69 @@ func TestExpandPath(t *testing.T) {
 		})
 	}
 }
+
+func TestNoAuthUser(t *testing.T) {
+	confFileName := createConfFile(t, []byte(`
```

Comment:
Maybe also use a test with code configuration, which if you do the check on validateOptions should catch it.

Reply:
ok, will add

Reply:
@kozlovic , when I moved the check to validateOptions, the test started to fail in a different location. (server start, which is why the error check was moved. After that it seemed pointless to generate the options in code as reading from file produced the same result